spec/lib/decision_review/service_spec.rb

```diff
@@ -9,232 +9,121 @@
 
-  describe '#post_higher_level_reviews' do
-    context 'with a valid decision review request' do
-      it 'returns an intake status response object' do
-        VCR.use_cassette('decision_review/202_intake_status') do
-          request = {
-            'data' => {
-              'type' => 'HigherLevelReview',
-              'attributes' => {
-                'receiptDate' => '2019-07-10',
-                'informalConference' => true,
-                'sameOffice' => false,
-                'legacyOptInApproved' => true,
-                'benefitType' => 'compensation',
-                'veteran' => {
-                  'fileNumberOrSsn' => '123456789',
-                  'addressLine1' => '123 Street St',
-                  'addressLine2' => 'Apt 4',
-                  'city' => 'Chicago',
-                  'stateProvinceCode' => 'IL',
-                  'zipPostalCode' => '60652',
-                  'phoneNumber' => '4432924565',
-                  'emailAddress' => 'someone@example.com'
-                },
-                'claimant' => {
-                  'participantId' => '44444444',
-                  'payeeCode' => '10'
-                }
-              }
-            },
-            'included' => [
-              {
-                'type' => 'RequestIssue',
-                'attributes' => {
-                  'decisionIssueId' => 2
-                }
-              }
-            ]
-          }
-
-          response = subject.post_higher_level_reviews(request.to_json)
-          expect(response).to be_accepted
-          expect(response).to be_an DecisionReview::Responses::Response
+  describe '#create_higher_level_review' do
+    subject { described_class.new.create_higher_level_review(request_body: body.to_json, user: user) }
+
+    let(:body) { VetsJsonSchema::EXAMPLES['HLR-CREATE-REQUEST-BODY'] }
+
+    context '200 response' do
+      it 'returns a properly formatted 200 response' do
+        VCR.use_cassette('decision_review/HLR-CREATE-RESPONSE-200') do
+          expect(subject).to respond_to :status
+          expect(subject.status).to be 200
+          expect(subject).to respond_to :body
+          expect(subject.body).to be_a Hash
         end
       end
     end
 
-    context 'with a malformed decision review request' do
-      it 'returns a 400 error' do
-        VCR.use_cassette('decision_review/400_intake_status') do
-          expect(StatsD).to receive(:increment).once.with(
-            'api.decision_review.post_higher_level_reviews.fail', tags: [
-              'error:CommonClientErrorsClientError', 'status:400'
-            ]
-          )
-          expect(StatsD).to receive(:increment).once.with('api.decision_review.post_higher_level_reviews.total')
-          expect { subject.post_higher_level_reviews({}) }.to raise_error(DecisionReview::ServiceException)
-        end
+    context '422 response' do
+      let(:body) { {} }
+      let(:exception) do
```

Comment:
I found this use of let to convert the subject into an exception a bit confusing.  So I dug into a rspec matchers for expecting to raise_error, and found you can also check attributes.  So I think this context (and the other "bad response contexts) would be clearer if they were something like this:
```
context '422 response' do
      let(:body) { {} }

      it 'throws a DR_422 exception' do
        VCR.use_cassette('decision_review/HLR-CREATE-RESPONSE-422') do
          expect { subject }.to raise_error(
            an_instance_of(DecisionReview::ServiceException).and having_attributes(key: 'DR_422')
          )
        end
      end
    end
```

Reply:
this is pretty close to how i originally had it written, but, for some reason, it doesn't work (not entirely sure why)

Reply:
the raise_error bit works! i must have typoed originally, but looks like you can't use that matcher (have_attributes) when passing a block to expect.
considering that, i'm tempted to leave it as is (so i can easily test the exception)

Reply:
I don't understand your comment.  The code I wrote above I tested on your branch and it worked.  It also covers everything you are testing about the exception.

Reply:
Oh whoops! I was trying different things before i tested your snippet, and should have copied and pasted when i finally did. I was doing this:
```
    context '422 response' do
      let(:benefit_type) { 'apricot' }

      it 'throws a DR_422 exception' do
        VCR.use_cassette('decision_review/HLR-GET-CONTESTABLE-ISSUES-RESPONSE-422') do
          expect { subject }.to raise_error(DecisionReview::ServiceException).and have_attribute(key: 'DR_422')
        end
      end
    end
```
(for the bottommost 422 context (get contestable issues))
I've not seen 
```
raise_error(
  an_instance_of( x)
)
```
before
i'm used to:
```
raise_error SomeClass
```
nifty!

Reply:
it's beautiful now 😍